docs/hardware/devices/rak-wireless/index.mdx

```diff
@@ -42,3 +42,17 @@ RAK14001 RGB LED<br/>
 [RAK1906](./wisblock/peripherals?rakmodules=Sensors&sensors=RAK1906) Environment Sensor<br/>
 RAK12013 Radar Sensor<br/>
 RAK13800 Ethernet Module<br/>
+
+## RAK WisMesh Products
+Besides the above mentioned modular hardware system "WisBlock", that allows to pick all components you want on your own, RAKwireless also offers a few meshtastic devices that come ready-to-go.
+
+Those RAKwireless Meshtastic Devices (based on the above WisBlock components, and hence fully supported by Meshtatstic) are listed on their [Meshtastic Documentation Center page](https://docs.rakwireless.com/product-categories/meshtastic/).
+
+This includes devices for different use cases like (not a complete list, just a few examples):
+- the portable Meshtastic devices 
```

Comment:
Convert this from a list with sub bullet points to heading level 3's with lists underneath, to match the above section.